### PR TITLE
fix: deprecate the `getStakeActivation` RPC method

### PR DESCRIPTION
**Pull request by steveluscher**
This was deprecated in https://github.com/anza-xyz/agave/pull/69

### CODE REVIEW

**Comment by steveluscher** [comment]
* **#2864** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2864?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @steveluscher and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/library-legacy/src/connection.ts [hunk 1]
```diff
@@ -3640,6 +3640,8 @@ export class Connection {
 
   /**
    * Returns epoch activation information for a stake account that has been delegated
+   *
+   * @deprecated Deprecated in 1.18; will be removed in a future version.
```
Comment (buffalojoec) [inline packages/library-legacy/src/connection.ts:3644]: nit: Maybe explicitly mention this is Solana/Agave 1.18, not Web3.js 1.18?

**Review by buffalojoec — APPROVED** [review]

File: packages/library-legacy/src/connection.ts (hunk 1 above)
Comment (steveluscher) [reply packages/library-legacy/src/connection.ts:3644]: Ooh. I mean… yeah. I'd have to do that for all of the comments in this file. I'll do that in the next PR.

**Comment by steveluscher** [comment]
### Merge activity

* **Jun 27, 5:19 PM PDT**: @steveluscher started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2864).

**Pull request merged**